src/openrct2/scripting/ScTile.hpp

```diff
@@ -1013,6 +1032,7 @@ namespace OpenRCT2::Scripting
                 ctx, &ScTileElement::isAdditionBroken_get, &ScTileElement::isAdditionBroken_set, "isAdditionBroken");
             dukglue_register_property(
                 ctx, &ScTileElement::isAdditionGhost_get, &ScTileElement::isAdditionGhost_set, "isAdditionGhost");
+            dukglue_register_property(ctx, &ScTileElement::direction_get, &ScTileElement::direction_set, "direction");
```

Comment:
This is added under the "Footpath only" section, but applies to all element types, right? This should be moved up to be listed under "All".

Reply:
I missed that, I'll move it up.